skbio/metadata/intersection.pyx

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
fine with me. might be nice to package the interval tree stuff separately (still under MIT) for those who don't want the full, bx-python install.